taskcluster/app_services_taskgraph/loader/__init__.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I don't know what's going on in this file but assume it's fine.

Reply:
This file defines common functions for loaders. A loader is in charge of taking every task in the kind, applying job-defaults and finding what are the right upstream dependencies. 

If you want to know more about the architecture of taskgraph, you can have a look at: https://johanlorenzo.github.io/blog/2019/10/24/taskgraph-is-now-deployed-to-the-biggest-mozilla-mobile-projects.html. If you don't have much time, I recommend watching the 3-minute-video.

Reply:
Added a comment.